theme.json

```diff
@@ -275,13 +276,13 @@
                 "containers": ["listMost"]
             },
             {
-                "type": "SWP\\Bundle\\TemplatesSystemBundle\\Widget\\MenuWidgetHandler",
```

Comment:
it should work with class namespace and it's more descriptive this way.

Reply:
When I send request with these values to API endpoint it defaults to html widget, only with integers it works fine